opensearch-operator/Dockerfile.oracle

```diff
@@ -0,0 +1,37 @@
+# Build the manager binary
+FROM  ghcr.io/oracle/oraclelinux:8-slim as builder
+
+RUN microdnf update -y \
+    && microdnf -y install golang \
+    && microdnf clean all
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY main.go main.go
+COPY api/ api/
+COPY controllers/ controllers/
+COPY pkg/   pkg/
+COPY opensearch-gateway/   opensearch-gateway/
+RUN mkdir /license
+COPY LICENSE README.md SECURITY.md THIRD_PARTY_LICENSES.txt /license/
```

Comment:
I think this will add the files in the builder but not the main image? You may wanna do this in the actual base image or copy from builder in the base image, I'd say do the former